pwnlib/tubes/remote.py

```diff
@@ -99,7 +99,7 @@ def _connect(self, fam, typ):
 
                 sock = socket.socket(self.family, self.type, self.proto)
 
-                if timeout != None and timeout <= 0:
+                if timeout is not None and timeout <= 0:
```

Comment:
Overall I feel like these `!= None` and `is not None` changes are overkill and just introduce code churn but other than gut feeling I don't see anything wrong with it.